runner/runner.go

```diff
@@ -303,6 +303,9 @@ func (runner *Runner) getStartingState(
 ) string {
 	var currentState string
 
+	ticker := time.NewTicker(time.Duration(startTimeoutMultiplier * float32(params.StartInterval)))
```

Comment:
It looks like we need one-shot timer.  Do not understand why we need ticker here?